docs/content/Deployment/Platform-Kubernetes.md

```diff
@@ -0,0 +1,516 @@
+          name: cubestore-worker-1
+          ports:
+            - containerPort: 10001
+          resources: {}
+          volumeMounts:
+            - mountPath: /cube/data
+              name: cubestore-worker-1-hostpath0
+      restartPolicy: Always
+      volumes:
+        - hostPath:
+            path: /home/docker/conf/.cubestore
+          name: cubestore-worker-1-hostpath0
+status: {}
+```
+
+And its' corresponding service in `cubestore-worker-1-service.yaml`:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    io.kompose.service: cubestore-worker-1
+  name: cubestore-worker-1
+spec:
+  ports:
+    - name: '10001'
+      port: 10001
+      targetPort: 10001
+  selector:
+    io.kompose.service: cubestore-worker-1
+status:
+  loadBalancer: {}
+```
+
+## Set up reverse proxy
+
+In production, the Cube.js API should be served over an HTTPS connection to
+ensure security of the data in-transit. We recommend using a reverse proxy; as
+an example, let's assume we're running the Kubernetes cluster on a cloud
+provider such as AWS or GCP. We'll also assume we already have the SSL
+certificate and key available on our local filesystem.
+
+<!-- prettier-ignore-start -->
+[[info |]]
+| You can also use a reverse proxy to enable HTTP 2.0 and GZIP compression
+<!-- prettier-ignore-end -->
+
+First, we'll run the following to create a secret in Kubernetes:
+
+```bash
+kubectl create secret tls <NAME_OF_CERTIFICATE> --key <PATH_TO_KEY_FILE> --cert <PATH_TO_CERT_FILE>
+```
+
+Next, we'll create a new ingress rule in a file called `ingress.yml`:
+
+```yaml
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: ingress-cube-api
+spec:
+  tls:
+    - hosts:
+        - <HOST_FROM_CERTIFICATE>
+      secretName: <NAME_OF_CERTIFICATE>
+  rules:
+    - host: <HOST_FROM_CERTIFICATE>
+      http:
+        paths:
+          - path: '/'
+            pathType: Prefix
+            backend:
+              service:
+                name: cube-api
+                port:
+                  number: 4000
+```
+
+For more advanced configuration, or for platforms where ingress is manually
+managed, please refer to the Kubernetes documentation for [Ingress
+Controllers][k8s-docs-ingress-controllers].
+
+## Security
+
+### Use JSON Web Tokens
+
+Cube.js can be configured to use industry-standard JSON Web Key Sets for
+securing its API and limiting access to data. To do this, we'll define the
+relevant options on our Cube.js API instance and Refresh Worker deployments:
+
+<!-- prettier-ignore-start -->
+[[warning |]]
+| If you have cubes that use `SECURITY_CONTEXT` in their `sql` property, then
+| you must configure [`scheduledRefreshContexts`][ref-config-sched-ref-ctx] so
+| the refresh workers can correctly create pre-aggregations.
+<!-- prettier-ignore-end -->
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+...
+spec:
+  template:
+    ...
+    spec:
+      containers:
+        - env:
+          ...
+          - name: CUBEJS_JWK_URL
```

Comment:
Is it `JWK` or `JWT` here?
I honestly don't know. :smile:

Reply:
It definitely is JWK: 

![image](https://user-images.githubusercontent.com/1781985/130762145-57cc62f8-0d2d-4a5b-9bca-64ef80ce4c83.png)

From https://cube.dev/docs/reference/environment-variables#general